plugin-build/src/test/kotlin/io/sentry/android/gradle/transforms/MetaInfStripTransformTest.kt

```diff
@@ -31,20 +31,22 @@ class MetaInfStripTransformTest {
             tmpDir: TemporaryFolder,
             jarName: String = "test.jar",
             multiRelease: Boolean = true,
-            includeSupportedVersion: Boolean = false
+            includeSupportedVersion: Boolean = false,
+            mimicSignedJar: Boolean = false
         ): MetaInfStripTransform {
             val file = tmpDir.newFile(jarName)
             val jar = file.toJar(
                 multiRelease = multiRelease,
-                includeSupportedVersion = includeSupportedVersion
+                includeSupportedVersion = includeSupportedVersion,
+                mimicSignedJar = mimicSignedJar
             )
 
             whenever(provider.get()).thenReturn(RegularFile { jar })
 
             return TestMetaInfStripTransform(provider)
         }
 
-        private fun File.toJar(multiRelease: Boolean, includeSupportedVersion: Boolean): File {
+        private fun File.toJar(multiRelease: Boolean, includeSupportedVersion: Boolean, mimicSignedJar: Boolean): File {
```

Comment:
```suggestion
        private fun File.toJar(multiRelease: Boolean, includeSupportedVersion: Boolean, signed: Boolean): File {
```

Let's stick to the naming pattern (e.g. we could also say `mimicMultiRelease` but we just use `multiRelease`). I think it's fine to name them like this, because it's just for tests